ui/src/app/components/footer.tsx

```diff
@@ -0,0 +1,196 @@
+import {openZendeskWidget} from 'app/utils/zendesk';
+import { environment } from 'environments/environment';
+import * as React from 'react';
+
+const styles = reactStyles({
+  footerAnchor: {
+    color: colors.secondary
+  },
+  footerAside: {
+    color: colors.white,
+    fontSize: 10,
+    lineHeight: '18px',
+    alignSelf: 'center'
+  },
+  footerImage: {
+    width: '6rem',
+    height: '2rem'
+  },
+  footerSectionHeader: {
+    color: colors.white,
+    borderBottom: `1px solid ${colors.white}`,
+    width: '100%'
+  },
+  footerTemplate: {
+    width: '100%',
+    height: '6.5rem',
+    backgroundColor: colors.primary,
+    padding: '1rem'
+  },
+  workbenchFooterItem: {
+    width: '30%'
+  }
+});
+
+const FooterTemplate = ({style = {}, ...props}) => {
+  return <div style={{...styles.footerTemplate, ...style}} {...props}>
+    <FlexRow>
+      <FlexColumn style={{height: '4rem', justifyContent: 'space-between'}}>
+        <img style={styles.footerImage} src='assets/images/all-of-us-logo-footer.svg'/>
+        <img style={{...styles.footerImage, height: '1rem'}} src='assets/images/nih-logo-footer.png'/>
+      </FlexColumn>
+      <div style={{marginLeft: '1.5rem', width: '100%', marginTop: '0.5rem'}}>
+        {props.children}
+      </div>
+    </FlexRow>
+  </div>;
+};
+
+const FooterSection = ({style = {}, header, ...props}) => {
+  return <FlexColumn style={style}>
+    <div style={styles.footerSectionHeader}>{header}</div>
+    <div>{props.children}</div>
+  </FlexColumn>;
+};
+
+const gettingStartedUrl = 'https://aousupporthelp.zendesk.com/hc/en-us/categories/360002157352-Getting-Started';
+const documentationUrl = 'https://aousupporthelp.zendesk.com/hc/en-us/categories/360002625291-Documentation';
+const communityForumUrl = 'https://aousupporthelp.zendesk.com/hc/en-us/community/topics';
+const faqUrl = 'https://aousupporthelp.zendesk.com/hc/en-us/categories/360002157532-Frequently-Asked-Questions';
+
+interface WorkbenchFooterProps {
+  profileState;
+}
+
+const WorkbenchFooter = withUserProfile()(
+  class extends React.Component<WorkbenchFooterProps, {}> {
+    constructor(props) {
+      super(props);
+    }
+    render() {
+      return <FooterTemplate>
+        <FlexRow style={{justifyContent: 'space-between', width: '100%', height: '100%'}}>
+          <FooterSection style={styles.workbenchFooterItem} header='Quick Links'>
+            <FlexRow>
+              <FlexColumn style={{width: '50%'}}>
+                <StyledAnchorTag style={styles.footerAnchor} href='/'>Home</StyledAnchorTag>
+                <StyledAnchorTag style={styles.footerAnchor} href='library'>Featured Workspaces</StyledAnchorTag>
+                <StyledAnchorTag style={styles.footerAnchor} href='workspaces'>Your workspaces</StyledAnchorTag>
+              </FlexColumn>
+              <FlexColumn style={{width: '50%'}}>
+                <StyledAnchorTag style={styles.footerAnchor} href={environment.publicUiUrl} target='_blank'>
```

Comment:
Do you think it will make a different to extract StyledAnchorTag out and have a const that takes in href variable and children text. It might not be much since its just one tag however its been used so many times i am just curious

Reply:
Do you mean another wrapper around StyledAnchorTag? Maybe. I can add it.

Reply:
Yeah thanks!